jmbitcoin/jmbitcoin/secp256k1_main.py

```diff
@@ -5,14 +5,10 @@
 import re
 import sys
 import base64
-import secp256k1
+import coincurve as secp256k1
```

Comment:
Is there a good reason to retain the `secp256k1` name?

Reply:
Mostly to minimize the diff, and coincurve still uses libsecp256k1 so I decided to leave it as is.